recipes/wasabi/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "wasabi" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a2ba97e2d960a25cba876806501f2dc68f74470ccef324c0d7aaf82a25a7422 
+build:
+  noarch: python
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+    requires:
+        - pytest <4
```

Comment:
I think this dependency can be dropped or you can run full test suite using something like [this](https://github.com/conda-forge/pyopencl-feedstock/blob/master/recipe/meta.yaml#L51-L54) (assuming it's relatively quick).

Reply:
> Looks good 👍 I have some minor suggestions and then once @ines confirms I'll merge

Thanks a lot for your help, especially on a Sunday!

Reply:
Hm, not sure I follow? How will it work if we don't have pytest noted as a dependency? It's not in the requirements.txt. Is it implicit for conda-forge?

Reply:
Thanks a lot for your help, especially on a Sunday!

Reply:
No problem! 😄 

At the moment the only test that is being ran is `python -c 'import wasabi'`. That's probably enough to prove that the package isn't completely broken and (probably?) doesn't require `pytest` to be installed.

To be more rigorous you could run the full test suite like is done in the example I linked.

Both options are fine from my perspective so it's up to you if you want to remove `requires: - pytest <4` or add `commands: python -m pytest ...`.

Reply:
Ah, got it. Yes we do want the tests to run. Fixed.

Reply:
@chrisburr Thanks so much! And agreed, we definitely want to run the tests.